### DISCUSSION TITLE
Module '"@nextui-org/react"' has no exported member 'createTheme'.

### DISCUSSION THREAD

**Discussion by Slap-a-tha-Bass** [heroui-inc/heroui#1604] [Feedback]
I upgraded to version 2.1.12 and tried to create some custom themes. It seems this is a later version?

**Comment by jrgarciadev** [comment]
I'll check thanks!

**Comment by jrgarciadev** [comment]
@Slap-a-tha-Bass we have never exported a `createTheme` function, you need to use the `nextui` plugin to do so

https://nextui.org/docs/customization/create-theme

**Comment by Slap-a-tha-Bass** [comment]
I was looking [here](https://v1.nextui.org/docs/theme/customize-theme) at these docs?

**Comment by jrgarciadev** [comment]
That's V1 documentation, we removed it since V2 is built on top of TailwindCSS and not Stitches

**Comment by Slap-a-tha-Bass** [comment]
Ohhh. Thank you for the clarity! :)
